sg/path.go

```diff
@@ -24,6 +26,34 @@ func FromWorkDir(pathElems ...string) string {
 	return filepath.Join(append([]string{cwd}, pathElems...)...)
 }
 
+// FromOSCache reutrns the path relative to OS cache folder, and a boolean indicating
+// whether such folder exists or not.
```

Comment:
```suggestion
// whether such folder exists.
```

When using "whether", you should not add "or not" as it's implied